drivers/at_cmd/at_cmd.c

```diff
@@ -324,6 +420,7 @@ static int at_cmd_driver_init(struct device *dev)
 
 int at_cmd_init(void)
 {
+	sys_slist_init(&notif_handler_list);
```

Comment:
What will happen if this driver is automatically initialized? The the notif_handler_list will not be initialized?

Reply:
That's a good point, @pkchan this needs to be fixed.